perf.proto

```diff
@@ -89,23 +90,42 @@ message MetricsCounters {
 
 message MetricsTimers {
   google.protobuf.Duration duration = 1;
-  google.protobuf.Duration waiting = 2;
+  google.protobuf.Duration total = 2;
 }
 
-message MetricsState {
-  bool failed = 1;
+message MetricsGuages {
+  int64 state = 1;
   int64 workers = 2;
+  bool failed = 3;
 }
 
 message MetricsEvent {
   string id = 1;
   repeated MetricsPoint Event = 3;
 }
 
+message RollupValue {
+  string name = 1;
+  oneof value {
+    int64 int = 2;
+    float fl = 3;
+  }
+  int64 version = 4;
+}
+
+message Rollups {
+  repeated RollupValue default_stats = 1;
+  repeated RollupValue user_stats = 2;
```

Comment:
I merged these. rebase and you can make this go away

Reply:
I still see `DefaultStats` and `UserStats` after rebase